CHANGELOG.md

```diff
@@ -1,14 +1,35 @@
-x.x.x Release notes (yyyy-MM-dd)
+10.0.0 Release notes (2020-10-14)
 =============================================================
+NOTE: This is a unified release note covering all v10.0.0-alpha.X and v10.0.0-beta.X releases.
+
 NOTE: Support for syncing with realm.cloud.io and/or Realm Object Server has been replaced with support for syncing with MongoDB Realm Cloud.
 
 NOTE: This version uses the Realm file format to version 20. It is not possible to downgrade to earlier versions than v10.0.0-beta.13. Non-sync Realms will be upgraded automatically. Synced Realms can only be automatically upgraded if created with Realm JavaScript v10.0.0-beta.1 and above.
```

Comment:
> This version uses the Realm file format to version 20.
**Either:** This version bumps the Realm file format to version 20.
**Or:** This version uses the Realm file format version 20.

Reply:
this is a common comment across the SDKs. I am hesitant to change anything about it.

Reply:
fixed below

Reply:
It's different in cocoa, and it's not currently correct english.. Additionally there's also this comment: https://github.com/realm/realm-cocoa/pull/6856/files#r504297984 about not mentioning specific file formats.